# 传感器校正默认配置文件

# 日志级别
logging_level: "INFO"

# 参考坐标系
reference_frame: "world"
sensor_array_frame: "sensor_array_frame"

# 机械臂服务
arm_service_name: "/mag_device_arm/set_end_effector_pose"

# 数据采集
data_collector:
  input_topic: "/mag_sensor/data_mT"
  use_batch_topic: false
  batch_topic: "/mag_sensor/batch"

# 可视化
visualization:
  marker_topic: "/calibration_markers"
  enabled: true

